records/new_metadata/newmeta_spec.yaml

```diff
@@ -57,6 +57,23 @@ pub:
   type: pub
   edge_annotation: true
 
+upstream:
+  description: Number of postsynaptic densities on a neuron in a specific region. Use this to annotate has_presynaptic_terminal_in relationships.
+  type: int
+  edge_annotation: true
+
+downstream:
+  description: Number of downstream synaptic connections (PSDs postsynaptic to the neuron T-bars) on a neuron in some specified region.  Use this to annotate 'has_postynaptic_terminal_in' relationships.
+  type: int
+  edge_annotation: true
+
+Tbars:
+  description: Number of Tbars that a neuron has in some specified region. Use this to annotate 'has_postynaptic_terminal_in' relationships.
```

Comment:
'Number of **T-bars** **annotated** on a neuron has in **a specific** region. Use this to annotate 'has_**post**ynaptic_terminal_in' relationships.'

Isn't 'T-bar' most commonly used in the literature? FBbt and GO have 't-bar' and 'T-bar' respectively. Also pre/post again, T-bars mark presynaptic sites no?